pkgs/by-name/it/itch/package.nix

```diff
@@ -1,95 +1,110 @@
-{ lib
-, stdenvNoCC
-, fetchzip
-, fetchFromGitHub
-, electron
-, steam-run
-, makeWrapper
-, copyDesktopItems
-, makeDesktopItem
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+  fetchFromGitHub,
+  electron,
+  steam-run,
+  makeWrapper,
+  copyDesktopItems,
+  makeDesktopItem,
 }:
-stdenvNoCC.mkDerivation rec {
-  pname = "itch";
-  version = "26.1.9";
+stdenvNoCC.mkDerivation (
+  finalAttrs:
+  let
+    butler = fetchzip {
+      url = "https://broth.itch.zone/butler/linux-amd64/15.21.0/butler.zip";
+      stripRoot = false;
+      hash = "sha256-jHni/5qf7xST6RRonP2EW8fJ6647jobzrnHe8VMx4IA=";
+    };
 
-  src = fetchzip {
-    url = "https://broth.itch.ovh/itch/linux-amd64/${version}/archive/default#.zip";
-    stripRoot = false;
-    hash = "sha256-4k6afBgOKGs7rzXAtIBpmuQeeT/Va8/0bZgNYjuJhgI=";
-  };
+    itch-setup = fetchzip {
+      url = "https://broth.itch.ovh/itch-setup/linux-amd64/1.26.0/itch-setup.zip";
+      stripRoot = false;
+      hash = "sha256-5MP6X33Jfu97o5R1n6Og64Bv4ZMxVM0A8lXeQug+bNA=";
+    };
```

Comment:
Why is a `let .. in` really needed here?

Reply:
What do you mean with "here"?

Reply:
> What do you mean with "here"?

I had to do it locally to demonstrate to explain this, so I pushed a different reformatting commit.